src/Raven.Studio/typescript/viewmodels/database/settings/connectionStrings.ts

```diff
@@ -216,11 +218,16 @@ class connectionStrings extends viewModelBase {
             });
     }
     
-    private getTasksThatUseThisString(connectionStringName: string, taskType: Raven.Client.Documents.Operations.OngoingTasks.OngoingTaskType) : { taskName: string; taskId: number }[]{        
-        const tasksData = this.connectionStringsTasksInfo[connectionStringName];               
-               
-        return tasksData ? _.sortBy(tasksData.map((task) => { return { taskName: task.TaskName, taskId: task.TaskId }; }), 
-                                    x => x.taskName.toUpperCase()) : [];    
+    private getTasksThatUseThisString(connectionStringName: string, taskType: Raven.Client.Documents.Operations.OngoingTasks.OngoingTaskType): { taskName: string; taskId: number }[] {
+        if (this.connectionStringsTasksInfo[connectionStringName] == null)
```

Comment:
in typescript we put { around if every time

Reply:
done